src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -397,11 +388,8 @@ static void processActivity(const char *user_key)
 
 static void usage(void)
 {
-    log_error("Wrong usage. Please reconfigure in squid.conf.\n");
```

Comment:
Removed this line; it is misleading.
usage() gets invoked when the -h option is used; it's not a wrong usage.